src/components/data-table-v2/_data-table-v2-inline-edit.scss

```diff
@@ -56,7 +56,7 @@
   }
 
   select {
-    padding: 0.45rem 2.75rem 0.45rem 1rem;
+    padding: 0.45rem 2.75rem 0.45rem $spacing-md;
```

Comment:
@marijohannessen I didn't change any of these values here to match the spacing variables because I wasn't sure how to view/test this. Should we leave them as is?

Reply:
You can just leave them as is! I didn't even know this file existed, and we're probably going to override a lot of it when we work on the inline editing functionality.